front-end/packages/robustar/src/views/ImageAnnotation.vue

```diff
@@ -467,13 +477,17 @@ export default {
       this.$root.startProcessing(
         'The editing information of this image is being sent. Please wait...'
       );
-      const image_url = sessionStorage.getItem('image_url') || '';
       const image_height = sessionStorage.getItem('image_height');
       const image_width = sessionStorage.getItem('image_width');
```

Comment:
Shall we move `image_height` and `image_width` to root as well so that the logic is consistent? 

Some extra info:
Currently these are set in `setCanvasImage` function in `graphics.js` within TUI image editor package. We may want to access the canvas object and read these values in our package whenever we need them.

Reply:
I think it is temporarily unnecessary to include these 2 statuses in the global context since there is only one place that accesses them for now. Also, tui-image-editor has exposed a method called `getCanvasSize()` which could retrieve these values.

Reply:
OK this makes sense.